gnomad/utils/vcf.py

```diff
@@ -583,7 +594,7 @@ def add_as_info_dict(
             ] = f"Allele-specific {first_letter}{rest_of_description}"
 
         except KeyError:
-            raise ValueError(f"{field} is not present in input info dictionary!")
+            logger.warning(f"{field} is not present in input info dictionary!")
```

Comment:
Assuming this is so that you can see them all at once, which is great, but I still think we should error out so maybe add an error if any fields were missing, but print all that were missing

Reply:
I'm not sure we should error out here with how this function is designed, since it's searching for site fields in INFO_DICT to generate the AS-version of that annotation

Reply:
Ah, OK, yes that makes sense